edge-apps/rss-reader/index.html

```diff
@@ -1,110 +1,82 @@
+	<div class="main-container">
+		 <div class="primary-container">
+			<div class="primary-card">
+				<span class="content-heading" x-text="entries[0]?.title || 'Loading...'">Loading...</span>
+				<span class="rss-tittle">RSS Feed</span>
+				<span class="content-p" x-text="entries[0]?.content || 'Loading...'">Loading...</span>
+				<span class="content-timestamp content-timestamp-0">Loading...</span>
+			</div>
+			<div class="primary-card">
+				<span class="content-heading" x-text="entries[1]?.title || 'Loading...'">Loading...</span>
+				<span class="rss-tittle">RSS Feed</span>
+				<span class="content-p" x-text="entries[1]?.content || 'Loading...'">Loading...</span>
+				<span class="content-timestamp content-timestamp-1">Loading...</span>
+			</div>
+		</div>
 
-              <div class="loader-icon"></div>
-            </div>
-          </div>
-        </template>
+		<div class="primary-container">
+			<div class="primary-card">
+				<span class="content-heading" x-text="entries[2]?.title || 'Loading...'">Loading...</span>
+				<span class="rss-tittle">RSS Feed</span>
+				<span class="content-p" x-text="entries[2]?.content || 'Loading...'">Loading...</span>
+				<span class="content-timestamp content-timestamp-2">Loading...</span>
+			</div>
+			<div class="primary-card">
+				<span class="content-heading" x-text="entries[3]?.title || 'Loading...'">Loading...</span>
+				<span class="rss-tittle">RSS Feed</span>
+				<span class="content-p" x-text="entries[3]?.content || 'Loading...'">Loading...</span>
+				<span class="content-timestamp content-timestamp-3">Loading...</span>
+			</div>
+		</div>
 
-        <template x-if="!fetchError">
-          <div id="feeds-container" class="d-flex flex-column h-100">
-            <div id="grid" class="mt-4">
-              <template x-for="entry in entries">
-                <div class="feed-container">
-                  <h2 class="feed-title fw-bolder" x-text="entry.title"></h2>
-                  <div class="feed-description" x-text="entry.contentSnippet">
-                  </div>
-                  <p class="feed-date" x-text="entry.pubDate"></p>
-                </div>
-              </template>
-            </div>
-          </div>
-        </template>
-      </main>
 
-      <footer class="container">
-        <p>&copy; 2024 Screenly, Inc.</p>
-      </footer>
-    </div>
+		<div class="secondary-container">
+			<div class="row-container">
+				<div class="secondary-card">
+					<div class="secondary-card-number-container">
+						<div class="secondary-card info-card">
+							<img id="brand-logo" src="static/img/Screenly.svg" class="brand-logo" alt="Brand Logo" />
+							<span class="info-text">Powered by Screenly</span>
+						</div>
+					</div>
+				</div>
+			</div>
+			<div class="row-container">
+				<div class="secondary-card">
+					<div class="secondary-card date-card">
+						<span class="date-text">MAY</span>
+						<span class="date-number">26</span>
+					</div>
```

Comment:
Do we need to show the defaults here before replacement?

Reply:
Adding a default value is not mandatory but is useful when running an emulator without values. I can remove it if you'd like.

Reply:
Fixed, removed default values.